### PR TITLE
DOC-6601 TA C20230118

### PR DESCRIPTION
**Pull request by andf-crl**
Addresses: DOC-6601

- Adds TA C20230118
- Moves entire TA section to new left-nav loc: **Releases > Technical Advisories** to somewhat suggest applicability to both other same-level entries in this loc: CC and CRDB.
- Adjusts TA index page (listing) to now also reference new CC-applicability for TAs.
- Minor language adjustments

Staging

- [advisories/index.md](https://deploy-preview-16016--cockroachdb-docs.netlify.app/docs/advisories/index.html)

### CODE REVIEW

File: advisories/c20230118.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: Technical Advisory C20230118
+advisory: C20230118
+summary: CockroachDB Cloud users with the Developer role could perform SQL admin-like operations using a specific internal DB Console API
+toc: true
+affected_versions: Cockroach Cloud from November 2022 - January 2023
```
Comment (abhinavg6) [inline advisories/c20230118.md:6]:
```suggestion
affected_versions: CockroachDB Cloud from November 2022 - January 2023
```

**Review by abhinavg6 — COMMENTED** [review]
Just a minor nit

File: advisories/c20230118.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+CockroachDB Cloud users that have been granted the [Developer role](../cockroachcloud/console-access-management.html#developer) in a cloud organization could use a specific DB Console API for any cluster in the organization and perform SQL admin-like operations including reading from or writing to any table in the cluster. The mentioned API is not publicly documented and is exclusively used internally by a select set of pages in the CockroachDB Cloud Console that retrieve data by proxying to a cluster’s DB Console. 
+
+The problem poses a company insider risk within your CockroachDB Cloud organization and increases the probability of data exfiltration by users that have been assigned the Developer role but are not supposed to have pertinent access to data outside of their scope. This applies to any cluster of CockroachDB within a cloud organization, regardless of version.
```
Comment (mikeCRL) [inline advisories/c20230118.md:17]:
```suggestion
The problem poses a company insider risk within your CockroachDB Cloud organization and increases the probability of data exfiltration by users that have been assigned the Developer role and are not intended to have access to data outside of their scope. This applies to any cluster of CockroachDB within a cloud organization, regardless of version. 
```

**Review by mikeCRL — COMMENTED** [review]
Looks good. Minor suggestions.

**Comment by andf-crl** [comment]
Thanks for feedback both. Please LGTM so that I can merge if you approve! Thanks!

**Review by mikeCRL — APPROVED** [review]

**Review by abhinavg6 — APPROVED** [review]

**Comment by andf-crl** [comment]
Thank you both, @abhinavg6 watching your thread for latest.

**Comment by andf-crl** [comment]
@abhinavg6 ah and there it is. Merging.

**Pull request merged**